bankayma_account/data/mail_template_views.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <template id="qweb_template_edi_invoice">
+        <div style="margin: 0px; padding: 0px;">
```

Comment:
@toyrobot a few words on this, I only tacitly corrected some typos and such, but I think you should rewrite the html a bit

Reply:
with qweb you can also take advantage of the [t-call](https://www.odoo.com/documentation/16.0/developer/reference/frontend/qweb.html#calling-sub-templates) directive, which would allow you to define some corporate identity that all your qweb mail templates recycle

Reply:
and obviously you should avoid it as much as possible, but if some text fragments can't be translated sensibly to he, you'd do something like

```xml
<p t-if="lang == 'he_IL'" t-translation="off">
The he version of the paragraph
</p>
<p t-else="">
Other languages in English with possibly a translation
</p>
```